cmd/internal/cli/inspect_linux.go

```diff
@@ -257,40 +271,130 @@ var InspectCmd = &cobra.Command{
 	Example: docs.InspectExample,
 
 	Run: func(cmd *cobra.Command, args []string) {
-		// Sanity check
-		if _, err := os.Stat(args[0]); err != nil {
-			sylog.Fatalf("container not found: %s", err)
+		sandboxImage := false
+		a := []string{"/bin/sh", "-c", ""}
```

Comment:
var declaration should be as close to the first use as possible

Reply:
also about naming:
the shorter the name, the shorter is the var's lifecycle. 
Here you use `a` in about 200 lines, so the name should be more expressive.